src/test/java/com/odysseusinc/athena/service/concept/SolrConceptPhraseSearchTest.java

```diff
@@ -177,6 +177,7 @@ public void query_phraseWithFirstExactWord() throws Exception {
         assertEquals(
                 Arrays.asList(
                         "Stroke Myocardial Infarction Gastrointestinal Bleeding",
+                        "Stroke",
```

Comment:
This one is slightly suspicious. From my point of view "Storke" is the "WholePhrase" match so probably it should be the first.